PrototypeDesignPattern1/Circle.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace PrototypeDesignPattern1
+{
+    internal class Circle: IComponent
+    {
+        public double Radius {  get; set; }
```

Comment:
You are accepting the value in constructor than its better to mark it as private

Reply:
Updated code